seqr/fixtures/1kg_project.json

```diff
@@ -334,11 +334,36 @@
         "analysis_notes": "",
         "analysis_summary": "",
         "analysis_status": "Q",
+        "success_story": "Published with Gleeson and Reza (PMID 31668703)",
+        "success_story_types": ["C", "D"],
         "internal_analysis_status": null,
         "internal_case_review_notes": "<div><span style=\"text-decoration: underline;\"><span style=\"font-family: 'book antiqua', palatino, serif;\">case review notes for family 12</span></span></div>\n<ul>\n<li><span style=\"font-family: 'book antiqua', palatino, serif;\">note1</span></li>\n<li><span style=\"font-family: 'book antiqua', palatino, serif;\">note 2</span></li>\n<li><span style=\"font-family: 'book antiqua', palatino, serif;\">note 3</span></li>\n</ul>",
         "internal_case_review_summary": "<div><span style=\"font-family: 'courier new', courier, monospace;\"><strong>summary for family 12</strong></span></div>"
     }
 },
+{
+    "model": "seqr.family",
```

Comment:
I general I try not to bloat our test feature with more data if we can repurpose existing data. Why do you need a new family, instead of tweaking one that already exists?

Reply:
I was trying not to change the existing data because I don't know whether the other testcases are assuming the values of the data. But for this test, it must be safe to tweak the existing data because it just adds two extra fields. I will update the data.

Reply:
I remember the reason for adding a new family now: I need to trigger an error which is caused by a family without any individual for the discovery_sheet API. The new family has no individual. Do you think it is ok to keep it?

Reply:
Sure, but maybe make the family_id "no_individuals" or "empty_family" or something so we remember what its for in the future

Reply:
Okay, I rename the display name of the family.

Reply:
Please rename the family_id field as I requested. There is no reason it needs to be the same as the pk

Reply:
Changed the id to "no_individuals".